### PR TITLE
fix: primary nav bar UI changes

### PR DESCRIPTION
**Pull request by dhananjay0601**
# Description

Changed the security icon, reduced the width of the primary nav bar, ensured the icons are aligned horizontally center in the nav bar. 

Fixes   https://github.com/devtron-labs/dashboard/issues/768

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [X] Tested by hovering over the primary nav bar

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [X] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/common/helpers/Helpers.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { ClusterImageList, ImageList } from '../../ClusterNodes/types'
 import { ApiResourceGroupType, K8SObjectType } from '../../ResourceBrowser/Types'
 import { getAggregator } from '../../app/details/appDetails/utils'
 import { SIDEBAR_KEYS } from '../../ResourceBrowser/Constants'
+import { cursorTo } from 'readline'
```
Comment (shivani170) [inline src/components/common/helpers/Helpers.tsx:17]: please remove if not using

File: src/css/base.scss [hunk 2]
```diff
@@ -2443,6 +2443,14 @@ textarea,
     margin-bottom: 0px !important;
 }
 
+.mt-20 {
```
Comment (shivani170) [inline src/css/base.scss:2446]: No need to create these css you can directly use mb-20 or mt-20

File: src/components/common/helpers/Helpers.tsx [hunk 3]
```diff
@@ -149,6 +150,7 @@ export function useForm(stateSchema, validationSchema = {}, callback) {
         } else {
             setState({ ...newState })
         }
+
```
Comment (shivani170) [inline src/components/common/helpers/Helpers.tsx:153]: remove this extra line

Comment (shivani170) [reply src/components/common/helpers/Helpers.tsx:153]: Revert the whole file if there is no change

**Review by milindk8 — APPROVED** [review]

File: src/components/login/Login.tsx [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
-import { toast } from 'react-toastify'
+import { Icons, toast } from 'react-toastify'
 import { ServerErrors } from '../../modals/commonTypes'
 import { URLS, Host, DOCUMENTATION, TOKEN_COOKIE_NAME} from '../../config'
 import { Progressing, showError , getCookie} from '../common'
 import { LoginProps, LoginFormState } from './login.types'
 import { getSSOConfigList, loginAsAdmin } from './login.service'
 import './login.css'
 import { dashboardAccessed } from '../../services/service'
+import InfoColourBar from '../common/infocolourBar/InfoColourbar'
```
Comment (shivani170) [inline src/components/login/Login.tsx]: Remove unused components

**Review by shivani170 — APPROVED** [review]

**Pull request merged**